### PR TITLE
4414.dev

### PR DESCRIPTION
**Pull request by 4reeman**
https://redmine.devbranch.work/issues/4414

### CODE REVIEW

File: web/themes/custom/fifth_sub_bootstrap/src/scss/custom/components/first_region/_region_right_side_first.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.block-views-blockeditor-picks-block-1 {
+  @include float_picture_text(true);
+}
+.block-views-blockeditor-picks-block-4 {
+  @include float_picture_text(true);
+}
```
Comment (devbranch-vitaliy) [inline web/themes/custom/fifth_sub_bootstrap/src/scss/custom/components/first_region/_region_right_side_first.scss]:
```suggestion
.block-views-blockeditor-picks-block-1,
.block-views-blockeditor-picks-block-4 {
  @include float_picture_text();
}
```

File: web/themes/custom/fifth_sub_bootstrap/src/scss/custom/abstract/mixins.scss [hunk 2]
```diff
@@ -7,3 +7,22 @@
     }
   }
 }
+
+@mixin float_picture_text($circle:false) {
```
Comment (devbranch-vitaliy) [inline web/themes/custom/fifth_sub_bootstrap/src/scss/custom/abstract/mixins.scss:11]:
```suggestion
@mixin float_picture_text($circle:true) {
```

File: web/themes/custom/fifth_sub_bootstrap/src/scss/custom/general/_views_view_fields.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+.views-field-title {
+  line-height: 1.1;
+  @include rfs(1.0625rem);
+  font-weight: 700;
+  margin: 0.5rem 0 0 0;
+  a {
+    color: $title-default;
+  }
+}
+.views-field-uid, .views-field-changed, .views-field-created, .views-field-revision-timestamp, .views-field-changed-1 {
```
Comment (devbranch-vitaliy) [inline web/themes/custom/fifth_sub_bootstrap/src/scss/custom/general/_views_view_fields.scss]: This is not a critical issue, but it's better to split up the different cases (classes)
```suggestion
.views-field-uid,
.views-field-changed,
.views-field-created,
.views-field-revision-timestamp,
.views-field-changed-1 {
```

File: web/themes/custom/fifth_sub_bootstrap/templates/page.html.twig [hunk 4]
```diff
@@ -124,7 +124,7 @@
     </header>
     <div id="main-wrapper" class="layout-main-wrapper clearfix">
       {% block content %}
-        <div id="main" class="{{ container }}">
+        <div class="{{ container }}">
```
Comment (devbranch-vitaliy) [inline web/themes/custom/fifth_sub_bootstrap/templates/page.html.twig]: Don't almost understand, why did you remove an ID of the `<div>`?

**Review by devbranch-vitaliy — CHANGES_REQUESTED** [review]

File: web/themes/custom/fifth_sub_bootstrap/templates/page.html.twig (hunk 4 above)
Comment (4reeman) [reply web/themes/custom/fifth_sub_bootstrap/templates/page.html.twig]:
```suggestion
        <div id="main" class="{{ container }}">
```

File: web/themes/custom/fifth_sub_bootstrap/src/scss/custom/abstract/mixins.scss (hunk 2 above)
Comment (4reeman) [reply web/themes/custom/fifth_sub_bootstrap/src/scss/custom/abstract/mixins.scss:11]: the default value must be false

File: web/themes/custom/fifth_sub_bootstrap/src/scss/custom/components/first_region/_region_right_side_first.scss (hunk 1 above)
Comment (4reeman) [reply web/themes/custom/fifth_sub_bootstrap/src/scss/custom/components/first_region/_region_right_side_first.scss]:
```suggestion
.block-views-blockeditor-picks-block-1,
.block-views-blockeditor-picks-block-4 {
  @include float_picture_text(true);
}
```

**Review by devbranch-vitaliy — APPROVED** [review]
